doc/man7/provider-mac.pod

```diff
@@ -196,22 +196,30 @@ Can be used to get the MAC block size (if supported by the algorithm).
 =item "fips-indicator" (B<OSSL_MAC_PARAM_FIPS_APPROVED_INDICATOR>) <int>
 
 A getter that returns 1 if the operation is FIPS approved, or 0 otherwise.
-This may be used after calling the final function.  It may return 0 if
-"no-short-mac" are set to 0.  This option is used by the OpenSSL FIPS
```

Comment:
Line breaks are the best way to determine the single or double space between sentences.
I.e. if you're going to change it, make it a new line for both here.